docs/source/notebooks/02_Extract_geographical_watershed_properties.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>It's already open with rasterio, no ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Changed sentence to reflect this</p>